platforms/documentation/docs/src/docs/release/notes-template.md

```diff
@@ -71,6 +71,15 @@ The following are the features that have been promoted in this Gradle release.
 ### Example promoted
 -->
 
+## Preview features
```

Comment:
I think preview features is maybe the wrong phrase here. We already have something called "feature previews". Feature previews are a way we introduce breaking behavior and allow builds to opt into the new behavior early.

e.g., type-safe project accessors are both an incubating feature and enabled with a feature preview.  Not every incubating feature will have a feature preview and not every feature preview will align with an incubating feature. 

A preview feature isn't a part of our feature lifecycle (internal -> incubating -> public -> deprecated).  We have the "promoted features" section above which is supposed to signal something going from incubating -> public/stable.  This usually happens after a long period of no other changes, so there's nothing significant to report. 

I'm not sure if we really want/need an entirely separate section dedicated to incubating (or preview features). That would imply the other section can only be updates to existing stable features.  Maybe we just need to be clear when an update is about an incubating feature/API or use some kind of special header decoration, like we do in the DSL reference.